packages/flutter/lib/src/widgets/interactive_viewer.dart

```diff
@@ -82,7 +87,7 @@ class InteractiveViewer extends StatefulWidget {
     this.scaleFactor = 200.0,
     this.transformationController,
     required Widget this.child,
-  }) : assert(alignPanAxis != null),
```

Comment:
Can you keep this assert here for now?  Once alignPanAxis is removed then this assert should be removed.

Reply:
Added it again